Trivial patch to add the string.h includes so that newer glibc won't
  cause errors.  This doesn't fix the compiler warnings though.
Signed-off-by: Robby Workman <rworkman@slackware.com>

diff -Nur acpitool-0.5.orig/src/freq.cpp acpitool-0.5/src/freq.cpp
--- acpitool-0.5.orig/src/freq.cpp	2008-07-20 17:47:55.000000000 -0500
+++ acpitool-0.5/src/freq.cpp	2009-05-18 10:48:45.151565890 -0500
@@ -34,7 +34,7 @@
 #include <dirent.h>
 #include "acpitool.h"
 #include "freq.h"
-
+#include <string.h>
 
 using namespace std;
 
diff -Nur acpitool-0.5.orig/src/toshiba.cpp acpitool-0.5/src/toshiba.cpp
--- acpitool-0.5.orig/src/toshiba.cpp	2007-11-06 15:19:59.000000000 -0600
+++ acpitool-0.5/src/toshiba.cpp	2009-05-18 10:49:00.400887087 -0500
@@ -33,6 +33,7 @@
 #include <iomanip>
 #include <fstream>
 #include "toshiba.h"
+#include <string.h>
 
 using namespace std;
 
